.eslintrc

```diff
@@ -0,0 +1,31 @@
+root: true
```

Comment:
Nop, YAML files are meant to start with this: https://docs.ansible.com/ansible/YAMLSyntax.html#yaml-basics
In practice, almost all parsers will parse YAML files without `---` just fine, but while we are about respecting style and good practices, let's do it globally :-)

Thanks for adding!